src/test/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderSearchCrnListAPIIntegrationTest.kt

```diff
@@ -0,0 +1,134 @@
+package uk.gov.justice.hmpps.offendersearch.controllers
+
+import io.restassured.RestAssured.given
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType
+import uk.gov.justice.hmpps.offendersearch.dto.OffenderDetail
+
+class OffenderSearchCrnListAPIIntegrationTest : LocalstackIntegrationBase() {
+
+  @BeforeEach
+  fun setUp() {
+    loadOffenders(
+        OffenderReplacement(crn = "X00001"),
+        OffenderReplacement(crn = "X00003"),
+        OffenderReplacement(crn = "X00088", deleted = true),
+    )
+  }
+
+  @Test
+  fun `not allowed to do a search without COMMUNITY role`() {
+    given()
+        .auth()
+        .oauth2(jwtAuthenticationHelper.createJwt("ROLE_BINGO"))
+        .contentType(MediaType.APPLICATION_JSON_VALUE)
+        .body("[\"X00001\",\"X00002\"]")
```

Comment:
I think you can use the triple quote here to avoid escaping, e.g. `.body("""["X00001", "X00002"]""")`

Reply:
Thanks - that should come in useful in event testing where the escaping makes it unreadable